autoPyTorch/pipeline/components/setup/network_backbone/utils.py

```diff
@@ -93,7 +112,30 @@ def backward(ctx: typing.Any,
 
 
 def shake_get_alpha_beta(is_training: bool, is_cuda: bool
-                         ) -> typing.Tuple[torch.tensor, torch.tensor]:
+                         ) -> Tuple[torch.Tensor, torch.Tensor]:
+    """
+    The methods used in this function have been introduced in 'ShakeShake Regularisation'
+    Each method name is available in the referred paper.
+    Currently, this function supports `even-even`, `shake-even`, `shake-shake` and `M3`.
```

Comment:
I think this string needs to be modified. On the development branch we don't have these methods, only shake shake and shake drop